utils/release/bump-package.mjs

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env node
```

Comment:
This script contains the first half of the former `npm-publish-package.mjs`.

The main difference is that I added the `modifyPackageJson` function and I replaced `npmBumpVersion` in favor of manually modifying `package.json`.